### PR TITLE
Refactor/mys 216 move out com binds

### PR DESCRIPTION
**Pull request by shroomist**

### CODE REVIEW

File: src/app/communication-bindings.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import MainMessageBusCommunication from './communication/main-message-bus-communication'
+import FeatureToggle from './features/feature-toggle'
+import ConnectionStatusEnum from '../libraries/mysterium-tequilapi/dto/connection-status-enum'
+import type { BugReporter } from './bug-reporting/interface'
+import StartupEventTracker from './statistics/startup-event-tracker'
+import logger from './logger'
+import { UserSettingsStore } from './user-settings/user-settings-store'
+import Notification from './notification'
+
+function showNotificationOnDisconnect (
```
[the remark this answers is not in the document]
Comment (shroomist) [reply src/app/communication-bindings.js]: to me it does make sense to construct with only `communication` thingy.
do you think that's gonna be enough for now?

[the remark this answers is not in the document]
Comment (ignasbernotas) [reply src/app/communication-bindings.js]: Yeah, I think that'll be awesome! ❤️

File: src/app/mysterium-vpn-params.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import SyncCallbacksInitializer from './sync-callbacks-initializer'
+import { BugReporterMetrics } from './bug-reporting/bug-reporter-metrics'
+import FeatureToggle from './features/feature-toggle'
+import type { BugReporter } from './bug-reporting/interface'
+import Notification from './notification'
+import StartupEventTracker from './statistics/startup-event-tracker'
+import type { StringLogger } from './logging/string-logger'
+import CommunicationBindings from './communication-bindings'
+
+export type MysteriumVpnParams = {
```
Comment (ignasbernotas) [inline src/app/mysterium-vpn-params.js]: This is so horrible that it's beautiful 😄

File: src/libraries/function-looper.js [hunk 3]
```diff
@@ -117,13 +119,12 @@ class FunctionLooper {
     const loop = async () => {
       // eslint-disable-next-line no-unmodified-loop-condition
       while (this._running) {
-        this._currentExecutor = new ThresholdExecutor(this._func, this._threshold)
+        this._currentExecutor = new ThresholdExecutor(
+          this._func,
+          this._threshold,
+          (err) => this._errorSubscriber.notify(err))
```
Comment (ignasbernotas) [inline src/libraries/function-looper.js]: Could you move the last ")" down a line? Pretty pleaaase 😄

Comment (shroomist) [reply src/libraries/function-looper.js]: sure, I wasn't sure myself about the convention here

**Pull request merged**